Lib/enum.py

```diff
@@ -77,6 +77,9 @@ def __setitem__(self, key, value):
                     ):
                 raise ValueError('_names_ are reserved for future Enum use')
             if key == '_generate_next_value_':
+                # check if members already defined
+                if self._member_names:
+                    raise TypeError("_generate_next_value_ must be defined before members")
```

Comment:
Looks like the change broke the test at line #1104.   Maybe a new `auto_called` flag that starts as `False` and changes to `True` when `auto()` is used the first time, and this test changes to `if auto_called:`.

Reply:
I added `auto_called` as a instance attribute. Is that what I should have done in this situation?

Reply:
Yup.  :)